themes/socialbase/templates/layout/region--hero.html.twig

```diff
@@ -1,8 +1,8 @@
 {%
   set classes = [
-    'region--' ~ region|clean_class,
-    'container',
-  ]
+  'region--' ~ region|clean_class,
+  style == 'sky' ? '' : 'container',
```

Comment:
We should probably just move this to a preprocess hook of the `attributes` array. We may decide in the future that there are some other conditions than the sky style that we'd want to change this.